dictionaries/software-terms/src/coding-terms.txt

```diff
@@ -248,3 +248,12 @@ varstatement
 waitfor
 webextensions
 webserver
+abab
+abcd
+abcba
+abcde
+abcdef
+abcdefg
+abcdefgh
+aead
+deadbeef
```

Comment:
I would have put them in a separate file, these one are noise or test equivalent to lorem sum

So while I would leave them in the dictionary. I would put them in a separate file that will be appended to the dictionary

Because anyone looking at these words may think, WTF

That's the issue with dictionary entry. You don't have a comment explaining the reason or context that led to add it

So having coding-lipsum.txt, with a few header explaining the context.

Or, it could be simply a section in the file anyway. I remember now you support that

What are you thoughts? @Jason3S

Reply:
I raised the point because I started looking at the coding terms for my own project

https://github.com/ccoVeille/jargon/issues/6#issue-2367698297

Reply:
@ccoVeille,

You make a good point. It is possible to add comments after the word, these will be ignored.

These happen to be common testing / debugging terms. But, without a comment, it would be hard to know that.

Maybe I should rename `coding-terms`.

The challenge with the `software-terms` dictionary is that it includes different kinds of things:

- Software / Technology terms and concepts - `software-terms.txt`
- Names of Tools and Services - `software-tools.txt`
- Common variable / function / property names  - `coding-terms.txt`

Reply:
> You make a good point. It is possible to add comments after the word, these will be ignored.

So like that 

```
foo # meta
abcdef # hex or alphabetical
```

Reply:
> Maybe I should rename coding-terms.\n\nThe challenge with the software-terms dictionary is that it includes different kinds of things:\n\nSoftware / Technology terms and concepts - software-terms.txt\nNames of Tools and Services - software-tools.txt\nCommon variable / function / property names - coding-terms.txt

I agree, but then it's always a mess to know where to put a word, but it must be your daily nightmare with project 😅🤣

Reply:
> > You make a good point. It is possible to add comments after the word, these will be ignored.
> 
> So like that ?
> 
> ```
> foo # meta
> abcdef # hex or alphabetical
> ```
> 
> Great then

Yes, like that.

Reply:
See #3310 for follow up PR